float32[] state
int32[] goal_actions
---
int8 action